.travis.yml

```diff
@@ -22,10 +22,13 @@ before_install:
     - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - phpunit --self-update
     - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+    - npm install -g swagger-cli
```

Comment:
>Having a packages.json in the repository (that doesn't use any NPM dependency) has no sense

Well, with this we are. If we don't add it it's a hidden project dependency. Not required unless you are developing on this bit, but hidden still. Having a `packages.json` does not force you to do a `npm install` when you want to get started with the project, but at least it avoids people to have to look at Travis to figure out that they need `swagger-cli`.

>it's good to test the Swagger support with the latest available version of swagger-cli

In that case yeah we can drop the `npm-shrinkwrap.json` thing, which makes it even easier as unlike composer, shrinkwrap depends on the local node version installed